site/_data/releases.yaml

```diff
@@ -1,12 +1,12 @@
 stable:
-  name: Euwe-2
-  tag: euwe-2
-  filename: euwe-2
+  name: Euwe-3
+  tag: euwe-3
+  filename: euwe-3
 
 prerelease:
- name: Fine-1-Beta1
- tag: fine-1-beta1
- filename: fine-1-beta1
+ name: Fine-1-RC1
+ tag: fine-1-rc1
+ filename: fine-1-rc1
```

Comment:
I think the changes for Fine Beta1 -> RC1 should be done in #519 instead of this PR which is for Euwe-3.